website/content/en/status/report-2023-04-2023-06/ifapi.adoc

```diff
@@ -1,7 +1,7 @@
 === Network Interface API (IfAPI)
 
-Links: +
-link:https://wiki.freebsd.org/projects/ifnet[Original project page] URL: link:https://wiki.freebsd.org/projects/ifnet
+Link: +
```

Comment:
I think we usually entitle the section "Links" and I would keep it as usual even if there is just one link, but if you prefer to switch to singular it is fine.